functionnal_tests/cypress/integration/account/account_main_page.js

```diff
@@ -10,7 +10,7 @@ describe('account :: main_page', function () {
   })
   it('', function () {
     cy.get('#dropdownMenuButton').click()
-    cy.get('a.setting__link[href="/ui/account"]').click()
+    cy.get('a.menuprofil__dropdown__setting__link[href="/ui/account"]').click()
```

Comment:
You could use the opportunity to change `class` selector by `data-cy` selector

Reply:
added data-cy and describe + it proper description